src/components/Grid.js

```diff
@@ -5,9 +5,28 @@ import { StyledGrid } from "../styles/StyledGrid";
 
 function Grid({ grid }) {
   return (
-    <StyledGrid>
-      {grid.map((row) => row.map((cell, x) => <Square key={x} type={cell} />))}
-    </StyledGrid>
+    <div>
+      <StyledGrid>
+        {grid[0].map((row, x) => (
+          <Square key={x} type={row} />
+        ))}
+      </StyledGrid>
+      <StyledGrid>
+        {grid[1].map((row, x) => (
+          <Square key={x} type={row} />
+        ))}
+      </StyledGrid>
+      <StyledGrid>
+        {grid[2].map((row, x) => (
+          <Square key={x} type={row} />
+        ))}
+      </StyledGrid>
+      <StyledGrid>
+        {grid[3].map((row, x) => (
+          <Square key={x} type={row} />
+        ))}
+      </StyledGrid>
+    </div>
   );
```

Comment:
Why this change?

Reply:
so this is what we need to discuss. If you can css it to work the other way the I'm all for it, but this was the easiest way for me to get the grid set up how we needed it

Reply:
Yeah, I'll take some time and play around with it. It kina hacky to hard-code in the grid rows and I want to try and avoid that this time around :)

Reply:
Updated it to removed all the mess